src/services/email.service.js

```diff
@@ -0,0 +1,42 @@
+import { config } from 'dotenv';
+import sendgrid from '@sendgrid/mail';
+import Debug from 'debug';
+
+config();
+
+const debug = Debug('dev');
+
+const { SENDGRID_API_KEY, SENDGRID_EMAIL, SENDGRID_ACTIVATION_TEMPLATE } = process.env;
+const templates = { accountActivation: SENDGRID_ACTIVATION_TEMPLATE };
+
+
+/**
+     * Initialises the event .
```

Comment:
```suggestion
/**
 * Initialises the event
```
Please ensure you align JSDoc items properly. Also always leave a new line between JSDoc properties e.g

```javascript
/**
 * Initialises the event

 * @function

 * @param {Object} message - The message payload.
 * @param {String} type - the type of message which corresponds to the mail template
 * @param {Object | Array} data - can be a list for bulk sending. must contain email parameter

 * @return {promise} - Returns a promise that get resolcved to a success object
 */
```

Reply:
eslint enforces the indentation I used

Reply:
Can I see a screenshot?

Reply:
<img width="1552" alt="Screenshot 2019-07-30 at 12 23 48 PM" src="https://user-images.githubusercontent.com/34121552/62125635-98995d80-b2c5-11e9-91ca-94bcbd3f5b3c.png">

Screenshot of Jsdoc indentation and Eslint red flag